modules/vaos/app/controllers/vaos/available_appointments_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_dependency 'vaos/application_controller'
+
+module VAOS
+  class AvailableAppointmentsController < ApplicationController
+    def index
+      response = systems_service.get_facility_available_appointments(
+        facility_id, start_date, end_date, clinic_ids
+      )
+
+      render json: VAOS::AvailabilitySerializer.new(response)
+    end
+
+    private
+
+    def systems_service
+      VAOS::SystemsService.new(current_user)
+    end
```

Comment:
hmmm, this works, but we've been using `.for_user` elsewhere.

Reply:
What `.for_user` do over standard ruby initialization?

Reply:
Mostly a nitpick comment I guess, but most of the other services initialize with `self.for_user`